tests/myTest.spec.js

```diff
@@ -0,0 +1,30 @@
+import { test, expect } from "@playwright/test";
+test.describe("Testing", () => {
+  test("test1", async ({ page }) => {
```

Comment:
names: test1 and test2 say nothing to us in the report. I would suggest to add more meaningful names for the scenarioas